app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -29,6 +29,7 @@ function show_interface_modal(modal_content) {
   modal_window.modal({'show': true});
 
   modal_window.find('a[rel="popover-modal"]').popover({html: true});
+  $('fieldset.compute_attributes select').select2();
```

Comment:
1. I personally think this should apply to all select lists on Foreman, as it is very easy to use and handle.
2. Currently, because it is in ```host_edit_interfaces.js``` in the function ```show_interface_modal``` this will apply only to CR network interfaces.